src/openzaak/components/zaken/constants.py

```diff
@@ -87,6 +89,7 @@ class AardZaakRelatie(models.TextChoices):
             "Aan het bereiken van de uitkomst van de andere zaak levert de onderhanden zaak een bijdrage."
         ),
     )
+    overig = ("overig", mark_experimental("Overig"))
```

Comment:
The string passed to `mark_experimental` must explicitly be marked as translatable in order for it to be picked up by `makemessages`
```suggestion
    overig = ("overig", mark_experimental(_("Overig")))
```